rke/k8s_rke_system_images.go

```diff
@@ -8522,5 +8522,137 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			IngressBackend:            m("k8s.gcr.io/defaultbackend:1.4"),
 			MetricsServer:             m("gcr.io/google_containers/metrics-server-amd64:v0.3.1"),
 		},
+		// Enabled out of band post v2.6.9
+		"v1.22.16-rancher1-1": {
```

Comment:
There probably won't be 1.22, see more info here: https://github.com/rancher/hyperkube/pull/291#issuecomment-1308896775

Reply:
Disregard, 1.22.16 actually got released (https://github.com/kubernetes/kubernetes/releases/tag/v1.22.16)